curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/619b7fd56aa2253778bcf5f7.md

```diff
@@ -9,14 +9,20 @@ dashedName: step-48
 
 The <dfn>HSL</dfn> color model, or hue, saturation, and lightness, is another way to represent colors.
 
-The CSS `hsl` function accepts 3 values: a number from 0 to 360 for hue, a percentage from 0 to 100 for saturation, and a percentage from 0 to 100 for lightness.
+The CSS `hsl` function accepts 3 values: 
```

Comment:
This step is way too verbose. We want to figure out a way to reduce the amount of text here – not increase it.

Reply:
I removed the example about using `%` in the saturation and lightness value and got what the HSL values to what it was (I broke it into a list).

Is that enough? It's still a lot of texts cuz there's the need to explain it like that.

Reply:
Yes – this looks much better and much clearer.